### ISSUE TITLE
Problem adding Model to definitions

### ISSUE THREAD

**Issue by aarcro** [axnsan12/drf-yasg#84]
Using a simplified serializer for a list view with a limited set of fields does not get a model added to definitions. We get this error instead.

> Resolver error at paths./customers-search/.get.responses.200.schema.items.$ref
> Could not resolve reference because of: Could not resolve pointer: /definitions/SimpleCustomerSerializerV1 does not exist in document
> 

```
    @swagger_auto_schema(
        manual_parameters=[search_query_param],
        responses={
            status.HTTP_200_OK: openapi.Response(
                'Customers found.',
                serializers.SimpleCustomerSerializerV1(many=True)),
            status.HTTP_404_NOT_FOUND: 'No matching customers found.',
            status.HTTP_429_TOO_MANY_REQUESTS: 'Rate limit exceeded.',
            status.HTTP_500_INTERNAL_SERVER_ERROR: 'Internal error.',
            status.HTTP_504_GATEWAY_TIMEOUT: 'Request timed out.',
        })
    def list(self, request, *args, **kwargs):
```

If we remove the custom response for 200, the default behavior works as expected and does generate the model in definitions. The code is very hard to trace to figure out what is going on here.

**Comment by axnsan12** [comment]
Hello.

This issue should have been fixed in [version 1.4.5](https://github.com/axnsan12/drf-yasg/commit/6ea8711a1f5a291f94d5b0ce1d40d6d45596a8b0). Are you running the latest version from PyPI?

**Comment by aarcro** [comment]
Thank you! 1.4.5 is working as expected.
